### DISCUSSION TITLE
Finally smal demo of NanoELS + DRO

### DISCUSSION THREAD

**Discussion by vegetate7** [kachurovskiy/nanoels#176] [Show and tell]
Just a beginning yet.
Shown setting left/right stops at manual mode, moving in gearbox mode, backlash takeout.
Restarting gearbox mode require some additional operations: disconnect halfnuts, move carriage, reconnect halfnuts, stop mode, restart mode. But it will be way simple, when I'll finish ballscrew conversion.

[![NanoELS + DRO demo](http://img.youtube.com/vi/wIaeWsB7uOQ/0.jpg)](http://www.youtube.com/watch?v=wIaeWsB7uOQ "NanoELS + DRO demo")]

https://www.youtube.com/watch?v=wIaeWsB7uOQ

**Comment by kachurovskiy** [comment]
Congratulations on making the DRO integration work! Very nice.

If you're getting ballscrews then backlash is 0 and axes can only be moved with H4 anyway so DRO becomes less necessary.

Which use cases will be simplified or improved when all is done?

**Reply by vegetate7** [reply]
Ballscrew will be only on Z axis. I making it with fixed screw and rotating nut design. The Z manual flywheel will be functional and on it's usual place. So the lathe will have usual manual feel and functional, unless ELS modes activated. At least it's the plan :)
I.e. if I want to "touch the workpiece with tool" and feel it, It will be possible, and much more convinient, than operated by rotary encoders.

**Comment by Festivejelly** [comment]
I love this idea as I dont want to convert either axis to ballscrew as sometimes I enjoy manual machining.
